src/less/components/tooltip.less

```diff
@@ -1,12 +1,7 @@
 .tooltip {
-    position: relative;
-    display: inline-flex;
-    margin-left: 0.25rem;
-    background-color: transparent;
-    border: 1px solid transparent;
-    padding: 0.5rem;
-    border-radius: 2px;
-    cursor: pointer;
+    i {
+        margin-right: 0 !important;
+    }
```

Comment:
this `!important` doesn't feel right to me, it feels like forcing CSS

plus, this margin-right is only for the button's icon, not for the tooltip per se, so it would be better if this was removed from the tooltip styling (just like the rest o fthe `.tooltip` you're removing in this PR 🌟 )
Could we add `.m-0` to the `<i>` classnames, and this way we remove this rules for `<i>` from the tooltips npm package styles?